src/image_processing/processing.py

```diff
@@ -0,0 +1,131 @@
+import cv2
+import numpy as np
+
+from .helpers import crop_box
+from .inference import HandDetector, HandClassifier
+
+
+class ImageProcessor:
+    def __init__(self, enlargebox_px=15, queue_size=20, drawing_state_threshold=0.5,
+                 inactivity_std_dev_threshold=4, activity_std_dev_lower_threshold=15,
+                 activity_std_dev_upper_threshold=100):
+        self.hand_detector = HandDetector(confidence=0.6)
+        self.hand_classifier = HandClassifier()
+
+        self.enlargebox_pt = enlargebox_px
+        self.drawing_state_threshold = drawing_state_threshold
+        self.inactivity_std_dev_threshold = inactivity_std_dev_threshold
+        self.activity_std_dev_lower_threshold = activity_std_dev_lower_threshold
+        self.activity_std_dev_upper_threshold = activity_std_dev_upper_threshold
+
+        self.image_size = self.hand_detector.get_image_size()
+        self.path_image = np.zeros(shape=self.image_size, dtype=np.uint8)
+
+        self.last_class_predictions = []
+        self.last_box_predictions = []
+        self.queue_size = queue_size
+
+        self.drawing_state = False
+        self.drawing_points = []
+        self.finish_drawing = False
+        self.is_outlier = False
+
+    def process_img(self, frame):
+        boxes, img_resized, image_resized_boxes = self.hand_detector.predict(img=frame, should_draw_results=True)
+        boxes_images = crop_box(img_resized, boxes, enlargebox_px=self.enlargebox_pt)
+
+        if len(boxes_images) > 0:
+            if len(boxes_images) > 1:
+                # TODO: Handle it better
+                # If there's more than one hand, get right hand
+                # Right hand has minimum x value
+                right_hand_index = np.argmin([box[0] for box in boxes])
+                boxes_images = [boxes_images[right_hand_index]]
+                boxes = [boxes[right_hand_index]]
+
+            for box_image, box in zip(boxes_images, boxes):
+                prediction = self.hand_classifier.predict(box_image, should_preprocess_input=True)
+                box_middle = [box[0], box[1]]
```

Comment:
Are you sure it's box middle and not left and top ? HandDetector returns box as (left, top, width, height).
```suggestion
                box_middle = [box[0] + box[2]/2, box[1]+box[3]/2]
```
I think  it would be what you want

Reply:
Yup, my mistake